### PR TITLE
feat(AssetKitBlock): changes & improvements

### PR DESCRIPTION
**Pull request by Walkeezy**
Ticket: https://app.clickup.com/t/8677vvdnn

- [x] Show/hide thumbnails option (but always show thumbnails in edit mode)
- [x] Show/hide asset count option
- [x] Add option for a custom asset count font color
- [x] Fix default styles for title & description
- [x] Use image caption styles for asset count
- [x] Change asset display to `cover`
- [x] Use smaller image sizes for the asset thumbnails
- [x] Add inline editing of button "Download package"

### CODE REVIEW

File: packages/asset-kit-block/src/AssetKitBlock.tsx [hunk 1]
```diff
@@ -95,7 +113,14 @@ export const AssetKitBlock = ({ appBridge }: BlockProps): ReactElement => {
                         onClick={startDownload}
                         style={BlockStyles.buttonPrimary}
                     >
-                        Download package
+                        <RichTextEditor
```
[the remark this answers is not in the document]
Comment (Walkeezy) [reply packages/asset-kit-block/src/AssetKitBlock.tsx:116]: I tried, but with the serialized HTML we get in view mode we are loosing the text styles from the Button that is wrapping the text. So I'd rather keep just the regular RTE from fondue (it's a really stripped down version anyway, which shouldn't cause performance issues).

[the remark this answers is not in the document]
Comment (fulopdaniel) [reply packages/asset-kit-block/src/AssetKitBlock.tsx:116]: having several blocks on the same page might still cause issues I think. Maybe it is worth checking and adding some memoization if it is not working properly. I'll leave it up to you

[the remark this answers is not in the document]
Comment (ragi96) [reply packages/asset-kit-block/src/AssetKitBlock.tsx:115]: Adjusted it so you can use our own RTE

[the remark this answers is not in the document]
Comment (Walkeezy) [reply packages/asset-kit-block/src/AssetKitBlock.tsx:115]: @ragi96 awesome! thanks 🙌

File: packages/asset-kit-block/src/AssetKitBlock.tsx [hunk 2]
```diff
@@ -92,10 +109,16 @@ export const AssetKitBlock = ({ appBridge }: BlockProps): ReactElement => {
                                 status
                             ) || currentAssets.length === 0
                         }
-                        onClick={startDownload}
+                        onClick={isEditing ? undefined : startDownload}
                         style={BlockStyles.buttonPrimary}
```
Comment (ragi96) [inline packages/asset-kit-block/src/AssetKitBlock.tsx:113]: Hover isn't working, check the serializer of the ButtonNode to get a clue how it will work

Comment (Walkeezy) [reply packages/asset-kit-block/src/AssetKitBlock.tsx:120]: Will be fixed in a separate PR

Comment (ragi96) [reply packages/asset-kit-block/src/AssetKitBlock.tsx:120]: Own task for it

**Review by ragi96 — APPROVED** [review]

**Pull request merged**